OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.h

```diff
@@ -85,8 +79,8 @@ namespace OpenSim {
 
     @author Matt Millard
 */
-class OSIMACTUATORS_API MuscleFirstOrderActivationDynamicModel : public Object{
-OpenSim_DECLARE_CONCRETE_OBJECT(MuscleFirstOrderActivationDynamicModel, Object);
+class OSIMACTUATORS_API MuscleFirstOrderActivationDynamicModel : public ModelComponent{
+OpenSim_DECLARE_CONCRETE_OBJECT(MuscleFirstOrderActivationDynamicModel, ModelComponent);
```

Comment:
This could just be a component. In my mind, the main distinguishing feature between a Component and a ModelComponent is that you can't add Component's directly to a model. That is, ModelComponents belong to a model, and Components belong to ModelComponents. Do you want to allow people to add this component to a model as a MiscModelComponent, or do you want to require that this component belong to another component?